src/codeu/chat/client/commandline/Chat.java

```diff
@@ -66,6 +96,19 @@ public boolean handleCommand(String line) {
       return true;
     }
 
+    if ("version-check".equals(command)) {
+      System.out.println("Version number: " + context.Version());
+      return true;
+    }
+
+    if ("up-time".equals(command)) {
+      Time serverTime = context.getTime();
+      Time now = Time.now();
+      double difference = now.inMs() - serverTime.inMs();
+      System.out.println("Server has been running for: " + toSeconds(difference) + "seconds");
```

Comment:
Can you add a pretty print for the uptime?  Java libraries might be able to handle most of the work here.